### PR TITLE
media-fonts/fontawesome: update to 6.1.1

### PR DESCRIPTION
**Pull request by gbeliako**
Signed-off-by: Chaquen Beliakov Amaya <c@cgps.ch>

### CODE REVIEW

File: media-fonts/fontawesome/fontawesome-6.1.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit font
+
+DESCRIPTION="The iconic font"
+HOMEPAGE="https://fontawesome.com"
+SRC_URI="https://github.com/FortAwesome/Font-Awesome/archive/refs/tags/6.1.1.tar.gz -> ${P}.tar.gz"
```
Comment (juippis) [inline media-fonts/fontawesome/fontawesome-6.1.1.ebuild:10]:
```suggestion
SRC_URI="https://github.com/FortAwesome/Font-Awesome/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
```

File: media-fonts/fontawesome/fontawesome-6.1.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+		FONT_S="${S}/otfs" FONT_SUFFIX="otf" font_src_install
+	fi
+	if use ttf; then
+		FONT_S="${S}/webfonts" FONT_SUFFIX="ttf" font_src_install
+	fi
+}
+
+DEPEND=""
+RDEPEND="${DEPEND}"
+BDEPEND=""
```
Comment (juippis) [inline media-fonts/fontawesome/fontawesome-6.1.1.ebuild:30]:
```suggestion
```

**Review by juippis — COMMENTED** [review]
Hmm, the jump from 5 to 6 seems noticeable.
```
 FILES:-usr/share/fonts/fontawesome/Font Awesome 6 Brands-Regular-400.otf
 FILES:-usr/share/fonts/fontawesome/Font Awesome 6 Free-Regular-400.otf
 FILES:-usr/share/fonts/fontawesome/Font Awesome 6 Free-Solid-900.otf
 FILES:+usr/share/fonts/fontawesome/Font Awesome 5 Brands-Regular-400.otf
 FILES:+usr/share/fonts/fontawesome/Font Awesome 5 Free-Regular-400.otf
 FILES:+usr/share/fonts/fontawesome/Font Awesome 5 Free-Solid-900.otf
```
Does it break anything?

File: media-fonts/fontawesome/fontawesome-6.1.1.ebuild (hunk 1 above)
Comment (gbeliako) [reply media-fonts/fontawesome/fontawesome-6.1.1.ebuild]: The jump from v5 to v6 (as far as i can tell) only adds new icons as per the changelog:
https://fontawesome.com/v6/docs/changelog/

i've updated the suggestions, but the documentation is a bit unclear in https://wiki.gentoo.org/wiki/Basic_guide_to_write_Gentoo_Ebuilds in the Note: "Using the [${PN} style variables](https://devmanual.gentoo.org/ebuild-writing/variables/) inside SRC_URI is allowed, though this is not necessarily best practice"

Thanks,

**Review by juippis — APPROVED** [review]
Thanks! 

`PN` is an unchangeable variable in most cases, while `PV` means the version which changes between version bumps. So for easier maintainability you want to use `${PV}` so it follows the version, while `PN` won't change so it only makes copy-pasting harder.

**Pull request closed without merging**